src/components/ExchangeSearch.tsx

```diff
@@ -3,9 +3,9 @@ import { isEmpty } from 'lodash';
 import React, { ForwardRefRenderFunction, MutableRefObject, useCallback, useContext, useEffect, useMemo, useRef } from 'react';
 import LinearGradient from 'react-native-linear-gradient';
 import Animated, { Easing, useAnimatedStyle, useSharedValue, withRepeat, withTiming } from 'react-native-reanimated';
-import Spinner from '../../assets/chartSpinner.png';
-import DiscoverSheetContext from '../../screens/discover/DiscoverScreenContext';
-import { ClearInputDecorator, Input } from '../inputs';
+import Spinner from '@/assets/chartSpinner.png';
```

Comment:
you can get rid of `ExchangeSearch` since the `DiscoverSearchInput` has its own `ExchangeSearch` defined and you can just replace `ExchangeSearchHeight` that is used in `CurrencySelectionList`